src/settings.js

```diff
@@ -47,11 +47,14 @@ var INVOKE_RUN = 1; // Whether we will run the main() function. Disable if you e
 var NO_EXIT_RUNTIME = 0; // If set, the runtime is not quit when main() completes (allowing code to
                          // run afterwards, for example from the browser main event loop).
 var MEM_INIT_METHOD = 0; // How to represent the initial memory content.
-                         // 0: keep array literal representing the initial memory data
+                         // 0: embed a base64 string literal representing the initial memory data
                          // 1: create a *.mem file containing the binary data of the initial memory;
                          //    use the --memory-init-file command line switch to select this method
                          // 2: embed a string literal representing that initial memory data
-                         //    XXX this is known to have bugs on windows, see https://github.com/kripken/emscripten/pull/3326
+                         //    XXX this is known to have bugs on windows, see https://github.com/kripken/emscripten/pull/3326;
+                         //    due to mode 0's recent upgrade to space-efficient base64 encoding
+                         //    and the long-standing issues with this mode, this mode is now
+                         //    deprecated in favor of mode 0
```

Comment:
> I made this change after @kripken suggested here that I add the deprecation note, which I'd assumed was related to whatever compatibility issues had prevented it from replacing MEM_INIT_METHOD=0 up until now (in that base64 is "good enough" while having no such issues).

Oh thanks for pointing back to this, I had missed that line of conversation (and sorry for the back and forth). I think we should not deprecate `MEM_INIT_METHOD=2`, since it does give a more efficient encoding than base64. The "bug" with that mode that the comment refers to is that on Windows the Closure minifier is known not to be able to process such generated files, but if one is not using Closure (which has been more common due to Closure not being asm.js compliant, we use our own minifier), then `MEM_INIT_METHOD=2` mode is usable on Windows as well.